go.mod

```diff
@@ -16,6 +16,7 @@ require (
 	k8s.io/api v0.23.1
 	k8s.io/apiextensions-apiserver v0.23.1
 	k8s.io/apimachinery v0.23.1
+	k8s.io/apiserver v0.23.1
```

Comment:
Same comment in the CRD validator e2e suite: can we eliminate this dependency (maybe in a follow-up?)?